### ISSUE TITLE
io-test failing on CI

### ISSUE THREAD

**Issue by quantumsteve** [project-asgard/asgard#682]
```
io-tests is a Catch2 v3.1.0 host application.
Run with -? for options

-------------------------------------------------------------------------------
highfive interface to HDF5 - double
  check it writes the correct file
-------------------------------------------------------------------------------
/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10
...............................................................................

/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10: FAILED:
due to unexpected exception with message:
  Impossible to write buffer with dimensions n = 1into dataset with dimensions
  [ 1, 5].

-------------------------------------------------------------------------------
highfive interface to HDF5 - float
  check it writes the correct file
-------------------------------------------------------------------------------
/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10
...............................................................................

/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10: FAILED:
due to unexpected exception with message:
  Impossible to write buffer with dimensions n = 1into dataset with dimensions
  [ 1, 5].

-------------------------------------------------------------------------------
highfive interface to HDF5 - int
  check it writes the correct file
-------------------------------------------------------------------------------
/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10
...............................................................................

/codebuild/output/src1627854762/src/github.com/project-asgard/asgard/src/io_tests.cpp:10: FAILED:
due to unexpected exception with message:
  Impossible to write buffer with dimensions n = 1into dataset with dimensions
  [ 1, 5].

===============================================================================
test cases: 3 | 3 failed
assertions: 3 | 3 failed



97% tests passed, 1 tests failed out of 29

Total Test time (real) =  84.11 sec
```
Neither @mkstoyanov or I can reproduce this locally so it may be related to the container or a change to the AWS CodeBuild environment.

**Comment by mkstoyanov** [comment]
Most likely a container issue.

Are they using standard containers that we can build locally (with docker file) or they have something AWS specific?

**Comment by quantumsteve** [comment]
https://github.com/project-asgard/containers/blob/main/cpu/Dockerfile

**Comment by mkstoyanov** [comment]
fixed in #688
